server/src/test/java/io/crate/integrationtests/PgCatalogITest.java

```diff
@@ -95,6 +95,18 @@ public void testPgConstraintTable() {
             "NULL| false| false| NULL| a| NULL| NULL| s| 0| a| 0| 0| true| NULL| t1_pk| -2048275947| true| NULL| NULL| 728874843| NULL| p| 0| true| -874078436\n"));
     }
 
+    @Test
+    public void testPgTablesTable() {
+        execute("select * from pg_tables WHERE tablename = 't1'");
+        assertThat(printedTable(response.rows()), is("false| false| false| false| doc| t1| NULL| NULL\n"));
```

Comment:
If it's not too much, I'd ask to add a test:
```
execute("select * from pg_tables WHERE tablename = 'v1'");
```
or with `count(*)` and assert 0 rows.